### PR TITLE
Test api routes

### PR DESCRIPTION
**Pull request by mathildemichau**
- Rename module name: fiches-urgence
- **fiches_urgence/config.py**: change storage folder for databases. Now stored into /fiches-urgences/db_instances/
**.gitignore**: Add ignore for db_instances folder
- **fiches_urgence/__init__.py**: 
set_sqlite_pragma: force sqlite to check foreign key constraints
create_app: construct instance of application within a context
- **fiches_urgence/main.py**: extract the main part of the module

- **tests/**: add Tests for API routes and its config
**fiches_urgence/tests/config_test.py**: 
Create a parent class with setUp and tearDown methosd to be used for all Child tests.
For this ,use another instance of app and  another instance of db
Add is_dict_subset_of_superset function to facilitates the tests
**test_*.py**: API tests for all existing routes

- Corrections following tests: 
fiches_urgence/models.py
fiches_urgence/routes.py

### CODE REVIEW

File: fiches_urgence/config.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+class Config(object):
+    # ...
```
Comment (Ibi-A) [inline fiches_urgence/config.py:5]: Useless comment or missing docstring!

File: fiches_urgence/routes.py [hunk 2]
```diff
@@ -217,7 +223,10 @@ def contributor_collection() -> utils.Response:
     if request.method == "GET":
         return get_collection(Contributor, contributors_schema)
     if request.method == "POST":
-        return create_new_item(Contributor, contributor_schema)
+        try:
+            return create_new_item(Contributor, contributor_schema, None, False)
+        except IntegrityError as err:
+            return "id attribute should be an existing person id", 400
```
Comment (Ibi-A) [inline fiches_urgence/routes.py:229]: The error message should be a JSON response as for the resident_collection integrity error.

File: fiches_urgence/config.py (hunk 1 above)
Comment (mathildemichau) [reply fiches_urgence/config.py]: Yep j'ai supprimé

File: fiches_urgence/routes.py (hunk 2 above)
Comment (mathildemichau) [reply fiches_urgence/routes.py]: ok, j'ai repris la methode http_response de utils

**Pull request merged**